openshift/4.0/pipelines/Jenkinsfile.cicd

```diff
@@ -130,24 +130,24 @@ pipeline {
       }
     }
 
-    stage('Force Build?') {
-      when {
-        expression { new Boolean(env.SKIP_BUILD) == true }
-      }
-      steps {
-        script {
-          try {
-            timeout(time: 2, unit: 'MINUTES') {
-              input(message: 'No code changes detected. Should we force the Build?', ok: 'Yes, we should')
-            }
-            env.SKIP_BUILD = 'false'
-          } catch (err) {
-            echo '[ci/cd]  No code changes. Skipping the Build stage...'
-            env.SKIP_BUILD = 'true'
-          }
-        }
-      }
-    }
+    // stage('Force Build?') {
```

Comment:
not sure if removing this in addition to the sonarqube changes was intentional or not.

Reply:
I'm thinking this causes us more grief than it's worth.  I'd rather just have the pipeline run if we start it and not ask the question.